doc/introduction/remote.rst

```diff
@@ -0,0 +1,511 @@
+.. _starship:
+
+Xanadu cloud platform
+#####################
+
+In this section, we provide a tutorial of the **RemoteEngine**, an engine used to connect to the Xanadu
+cloud platform and execute jobs remotely (e.g., on a quantum chip).
+
+Configuring the RemoteEngine
+----------------------------
+
+Before using the RemoteEngine, you need to configure the hostname and authentication token that will provide
+you access to the API. This can be done in one of two ways:
+
+1. By using the :func:`~.store_account` function to store your account credentials:
+
+   >>> sf.store_account("my_api_token")
```

Comment:
Included suggestion for consistency previously, not sure if we would like to make it consistent for the entire doc on `"my_api_token"` (if so, would need a quick search&replace)

Reply:
Also, on the `Configuration` page, this order (from Python & from CLI) was reverse, starting with the CLI example. Would we like to have it like that here as well, or is the change of order intended?

Reply:
I think the order is fine :) Not intended, but not too much of an issue.

But it would be good to standardize the dummy name we use for the tokens... which one do you prefer?

Reply:
Any of the ones that appeared work for me! Left the suggestions for potential convenience, saw the further cases later.